### PR TITLE
EMT-1 Add fix for handling comments

### PR DESCRIPTION
**Pull request by malminhas**
[EMT-1](https://karhoo.atlassian.net/browse/EMT-1)
* Add support for optional retrieval of comments from `engineering_metrics`

### CODE REVIEW

File: engineeringmetrics/adapters.py [hunk 1]
```diff
@@ -34,14 +35,8 @@ def busday_duration(date_a: datetime, date_b: datetime = None, interval="hours")
...
     bus_days = np.busday_count(
         date_a.date(), date_b.date()).item()
-    duration = full_duration
-
-    if full_duration.days == 2 and bus_days == 1:
-        duration = full_duration - timedelta(days=(2))
-    elif full_duration.days > bus_days:
-        duration = full_duration - \
-            timedelta(days=(full_duration.days - bus_days))
-
```
Comment (russormes) [inline engineeringmetrics/adapters.py:44]: Why did you remove this? This is to deal with some edge cases around weekends.

Comment (malminhas) [reply engineeringmetrics/adapters.py]: Whoops.  This is unintended probably the result of a stash-pull-unstash as I was on an older `engineering_leads`.  I certainly didn't intend to change that code.  Well spotted :-/.

Comment (malminhas) [reply engineeringmetrics/adapters.py]: Fixed in commit [3f90c56](https://github.com/karhoo/engineering-metrics/pull/32/commits/3f90c56877ea42fa44f9149f57991a9ef7b62c2a)

**Review by russormes — APPROVED** [review]
